server/routes/api/index.js

```diff
@@ -1,8 +1,11 @@
 import express from 'express';
 import userRoutes from './users';
+import Token from '../../helpers/Token';
```

Comment:
Don't use complex relative paths (that need you going back .

This should have been 
`import Token from '@helpers/Token';`

Reply:
Ok. Thanks